### PR TITLE
pythonPackages.pushover: init at 0.3

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Motivation for this change

Needed for home-assistant.

Cc: @FRidh @dotlambda 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by FRidh — APPROVED** [review]

File: pkgs/development/python-modules/pushover/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0xlghiqd9rsgn7jdhc8v1xh3xspssihrw1vyy85gvjzxa1ah19sk";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  # checks crash
+  doCheck = false;
```
Comment (dotlambda) [inline pkgs/development/python-modules/pushover/default.nix]: Tests could be run using `python test.py` but they require network access: https://github.com/laprice/pushover/blob/master/test.py

File: pkgs/development/python-modules/pushover/default.nix [hunk 2]
```diff
@@ -12,7 +12,7 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ requests ];
 
-  # checks crash
+  # there are no tests
```
Comment (dotlambda) [inline pkgs/development/python-modules/pushover/default.nix]: @Mic92 That's not true: https://github.com/laprice/pushover/blob/master/test.py

Comment (Mic92) [reply pkgs/development/python-modules/pushover/default.nix]: fixed: c3535901c3b64e244604be89b144a0e3330525f9

Comment (dotlambda) [reply pkgs/development/python-modules/pushover/default.nix]: @Mic92 Thanks!

**Pull request merged**